caasp-istio-proxyv2-image/caasp-istio-proxyv2-image.kiwi

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: caasp/v4/istio-proxyv2:1.5.4 -->
```

Comment:
Please use the three tags approach. Also note that this tag is redundant, you already defined that one in line 17.

Reply:
Noted. Will be updated in the next push. Thanks.